kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -13,24 +13,47 @@
       <mat-svg v-if="isFullscreen" class="icon" category="navigation" name="fullscreen_exit"/>
       <mat-svg v-else class="icon" category="navigation" name="fullscreen"/>
     </icon-button>
-    <div ref="pdfcontainer" class="pdfcontainer"></div>
+    <div ref="pdfcontainer" class="pdfcontainer" @scroll="checkPages">
+      <progress-bar v-if="loading" class="progress-bar" :show-percentage="true" :progress="progress"/>
+      <p class="page-container" v-for="index in totalPages"
+        :ref="pageRef(index)"
+        :style="{ height: pageHeight + 'px', width: pageWidth + 'px' }">
```

Comment:
Add a `<p>` tag for each page, and set the size of each to the size of a page. This will make the scroll window the right height.